### PR TITLE
[MODFQMMGR-487] Simple POL entity adjustments

### PR DESCRIPTION
**Pull request by ncovercash**

### CODE REVIEW

**Review by mweaver-ebsco — APPROVED** [review]

**Review by bvsharp — APPROVED** [review]

File: src/main/resources/entity-types/orders/simple_purchase_order_line.json5 [hunk 1]
```diff
@@ -858,6 +876,7 @@
               dataType: {
                 dataType: 'stringType',
               },
+              essential: true,
```
Comment (kjain110) [inline src/main/resources/entity-types/orders/simple_purchase_order_line.json5:879]: we don't support essential over here

Comment (ncovercash) [reply src/main/resources/entity-types/orders/simple_purchase_order_line.json5:879]: kathleen said to put it there 🙄

Comment (kjain110) [reply src/main/resources/entity-types/orders/simple_purchase_order_line.json5]: I don't think it will work

Comment (ncovercash) [reply src/main/resources/entity-types/orders/simple_purchase_order_line.json5]: i removed

**Review by kjain110 — APPROVED** [review]

File: src/main/resources/entity-types/orders/simple_purchase_order_line.json5 [hunk 2]
```diff
@@ -248,6 +253,7 @@
       isIdColumn: false,
       queryable: false,
       visibleByDefault: false,
+      essential: false,
```
Comment (kjain110) [inline src/main/resources/entity-types/orders/simple_purchase_order_line.json5:256]: by default it's set to false, I don't think you need to explicitly mention it.

File: src/main/resources/entity-types/orders/simple_purchase_order_line.json5 [hunk 3]
```diff
@@ -322,6 +328,7 @@
               valueGetter: "( SELECT array_agg(elems.value->>'contributor') FROM jsonb_array_elements(:sourceAlias.jsonb->'contributors') AS elems)",
               filterValueGetter: "( SELECT array_agg(lower(elems.value->>'contributor')) FROM jsonb_array_elements(:sourceAlias.jsonb->'contributors') AS elems)",
               valueFunction: 'lower(:value)',
+              visibleByDefault: true,
```
Comment (kjain110) [inline src/main/resources/entity-types/orders/simple_purchase_order_line.json5:331]: This will cause the PR to fail.

Comment (ncovercash) [reply src/main/resources/entity-types/orders/simple_purchase_order_line.json5]: yep, fixed it, thanks

**Pull request merged**